psi4/src/psi4/libmints/gshell.h

```diff
@@ -264,7 +264,9 @@ class PSI_API GaussianShell {
 
     /// Returns the center of the Molecule this shell is on
     const double* center() const;
-    /// Returns the atom number this shell is on. Used by integral derivatives for indexing.
+    /// Returns the n'th coordinate of the center of the Molecule this shell is on
```

Comment:
```suggestion
    /// Returns the icoord'th coordinate of the center of the Molecule this shell is on
```
or the reverse if you prefer. allowed vals 0, 1, 2?

Reply:
Yeah, the allowed values are 0, 1, and 2. Guessing that should be specified in the comment as well?